lib/tabs/hourtable/pages/plan_display.dart

```diff
@@ -157,6 +157,11 @@ class StuPlanDisplayState extends State<StuPlanDisplay> {
   late DateTime startDate;
   final _ctr = StuPlanDayDisplayController();
 
+  /// Lädt die Daten aus dem Cache neu - nur verwenden, wenn es schon Daten im Cache gibt
```

Comment:
```suggestion
  /// Lädt die Daten neu
```
Die Funktion ruft irgendwann `IndiwareDataManager.get[Kl/Le]DataForDate` auf, und der fragt das einfach vom Server ab falls das noch nicht im Cache liegt.